compatible-ide.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "displayName": "IntelliJ IDEA Community",
+    "dockerImage": "che-idea",
+    "productCode": "IC",
+    "productVersion": [
+      {
+        "version": "2020.3.3",
+        "downloadUrl": "https://download.jetbrains.com/idea/ideaIC-2020.3.3.tar.gz",
```

Comment:
Not related to this PR, side question
Do we plan to use 2021 binaries soin ?

Reply:
2021 is not supported yet by Projector Server, there are some incompatibility with Markdown plugin AFAIK.
There is an upstream issue for that https://youtrack.jetbrains.com/issue/PRJ-547#focus=Comments-27-4967802.0-0